README.md

```diff
@@ -124,6 +124,35 @@ If you want to check what resources are going to be targeted without actually te
 cloud-nuke aws --resource-type ec2 --dry-run
 ```
 
+### Ignoring list/nuke errors for certain/all resources
+
+Ideally you should have AWS IAM permissions to list and nuke all target resources. If you do not have permissions to
+nuke resource X - you should exclude it using the `--exclude-resource-type` flag. However, if you don't know which resource
+types you are not permitted to nuke or you have list access for all but nuke access for some, you can ignore their
+errors by specifying the `--ignore-errors` flag:
+
+```shell
+cloud-nuke aws --ignore-errors ec2 --ignore-errors s3
```

Comment:
We still need to provide users with the a pointer to what resource types can be specified via `--ignore-errors`. i.e., It's more than just `ec2` and `s3`, right?